apply/templates/apply/landing.html

```diff
@@ -53,7 +53,7 @@ <h1 class="govuk-heading-xl">{%trans fund_title=fund.title%}Start or continue an
             {% endif %}
 
             {% set availableInLanguageHtml%}
-                {% trans url=url_for('select_language', locale=('cy' if get_lang() == 'en' else 'en' )) %} This service is also available <a href="{{ url }}" class="govuk-link">in Welsh (Cymraeg){% endtrans %}</a>.
+                {% trans url=url_for('select_language', locale=('cy' if get_lang() == 'en' else 'en' )) %} This service is also available <a href="{{ url }}" target="_blank" class="govuk-link">in Welsh (Cymraeg){% endtrans %}</a>.
```

Comment:
As this touches generic translatable strings, can we please update the translation files as well? 🙏  PfN might not have Welsh text but another fund might, so good to fix this at the same time rather than have a weird Welsh-specific bug seem to pop up in the future.

Not urgent obviously, sort PfN go-live first, but let's please come back to this ASAP after 🙏